app/src/main/res/xml/changelog_master.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <changelog xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="MissingDefaultResource">
-    <release version="2025.7.1 - Main" versioncode="3">
+    <release version="2025.7.2 - Main" versioncode="3">
+        <change>Downloading snapshots from camera card is now working</change>
```

Comment:
@dshokouhi does it make sense to add this to automotive too?

Reply:
It technically applies to Automotive so I would also add it there